themes/bootstrap3/templates/Recommend/ConsortialVuFindDeferred.phtml

```diff
@@ -0,0 +1,9 @@
+<?php
+    // Set up Javascript for use below:
+    $loadJs = 'var url = VuFind.path + "/AJAX/Recommend?' . $this->recommend->getUrlParams() . '";'
+        . "VuFind.loadHtml(\$('#consortialVuFindDeferredRecommend'), url);";
+?>
+<div id="consortialVuFindDeferredRecommend">
```

Comment:
Obviously not something to do right here, but maybe a useful future project: I wonder if we should build a `deferredRecommend` view helper that builds this HTML and Javascript, since I'm pretty sure most of these deferred templates are identical apart from the container ID.

Reply:
I see you've opened https://github.com/vufind-org/vufind/pull/3283 to address this suggestion. Should we get that PR merged before finishing up this one, so it can be taken advantage of here?

Reply:
Yeah I think it's fine to wait, if we can get #3283 merged in the next day or two, which hopefully is likely.

Reply:
Yep, I'll see what I can do to help there...

Reply:
Okay, it's merged, so you're now free to simplify here, if possible...

Reply:
Done!